### DISCUSSION TITLE
Permission Errors on Linux - user #100999

### DISCUSSION THREAD

**Discussion by asim-munshi** [wardenenv/warden#695] [Help & Support]
Hi,

I have a fresh install of Warden on Linux.

Any files I generate/modify by logging into Warden SSH set an incorrect permission on my Host system. I have to reset permissions each time after making a modification.

Host System:
drwxr-xr-x 12 100999 dockerdesktop 4.0K Aug 29 22:57 exampleproject

**Comment by toonvd** [comment]
Hi

Check here for inspiration on how to solve this:

https://github.com/wardenenv/warden/issues/155
